package.json

```diff
@@ -23,22 +23,22 @@
   },
   "homepage": "https://github.com/amannn/action-semantic-pull-request#readme",
   "dependencies": {
-    "@actions/core": "^1.2.6",
+    "@actions/core": "^1.6.0",
     "@actions/github": "^5.0.0",
     "conventional-changelog-conventionalcommits": "4.5.0",
     "conventional-commit-types": "^3.0.0",
-    "conventional-commits-parser": "^3.2.0"
+    "conventional-commits-parser": "^3.2.4"
   },
   "devDependencies": {
     "@semantic-release/changelog": "5.0.1",
     "@semantic-release/commit-analyzer": "8.0.1",
     "@semantic-release/git": "9.0.0",
     "@semantic-release/github": "7.2.0",
-    "@semantic-release/release-notes-generator": "9.0.1",
+    "@semantic-release/release-notes-generator": "9.0.3",
     "@vercel/ncc": "^0.27.0",
-    "eslint": "7.9.0",
-    "eslint-config-molindo": "5.0.1",
-    "jest": "24.9.0",
+    "eslint": "8.8.0",
+    "eslint-plugin-node": "^11.1.0",
+    "jest": "27.4.7",
```

Comment:
Upgrading dependencies is fine, thanks! But please don't change the ESLint config.